### ISSUE TITLE
Demo does not work on IE 11

### ISSUE THREAD

**Issue by crutchcorn** [owsolutions/amazing-time-picker#54]
Upon trying to load the demo page in IE11, you'd get the following error message:
```
SCRIPT438: Object doesn't support property or method 'assign'
```

You could easily polyfill Object.assign to fix this with something like [this](https://www.npmjs.com/package/es6-object-assign)

**Comment by torabian** [comment]
Can you give us a hint how to fix this issue?

**Comment by crutchcorn** [comment]
#56 seems to fix this

**Comment by torabian** [comment]
Thanks to @yousefsami Now we also support IE 10. Available from version 1.5.3
